Github Issue Reader/Models/IssueModel.swift

```diff
@@ -6,17 +6,30 @@
 //
 
 import Foundation
+import UIKit
 
-struct Issue: Codable, Equatable {
+struct Issue: Codable, Hashable {
     let id: Int
-    let title: String?
+    let title: String
     let state: State
-    let user: User?
+    let user: User
     let body: String?
-    let createdAt: String?
-    
-    enum State: String, Codable, Equatable {
+    let createdAt: String
```

Comment:
From https://docs.github.com/en/rest/issues/issues#list-repository-issues it's a required property with this schema

```
"created_at": {
  "type": "string",
  "format": "date-time",
  "examples": [
    "2011-04-10T20:09:31Z"
  ]
},
```

Looks like one of the standard date formats so we definitely could. I haven't bothered messing with it because we don't actually need the field anywhere.

Reply:
👍